### PR TITLE
Render index markdowns for docs resources; add minimal repo-schema wi…

### PR DESCRIPTION
**Pull request by codingone21**
Rendering `index` paths for the repositories with /docs sources, other than /cadence, which is already configured.

UPDATES:
* Side bars and rendering for [flow-go-sdk, fcl-js, fcl-js-testing] are already merged.
* I noticed from the current navigation menus that we are also leading to 'dapp architecture' for which the content also lives in flow, so I've added that to the list of internal flow contents

**flow/../kitty-items**
There are some issues with loading image files. Below is the error log for reference.
```
Updating the cache value for flow:content/kitty-items/landing-page.png:compiled. Getting a fresh value for this took 239.863ms. Caching for a minimum of 30 days in redis.
Updating the cache value for flow:content/kitty-items/kitty-items-diagram.png:compiled. Getting a fresh value for this took 0.607ms. Caching for a minimum of 30 days in redis.
CatchBoundary { data: { status: 'noPage' }, status: 404, statusText: '' }
Updating the cache value for flow:content/kitty-items/kitty-items-diagram.png:compiled. Getting a fresh value for this took 258.382ms. Caching for a minimum of 30 days in redis.
GET /kitty-items/landing-page.png 404 - - 299.312 ms
CatchBoundary { data: { status: 'noPage' }, status: 404, statusText: '' }
```
Screen Shot 2022-06-20 at 10 46 14 PM

**flow/../vscode-extension**
Screen Shot 2022-06-20 at 10 48 17 PM

**flow/../dapp-development**
Screen Shot 2022-06-23 at 11 52 14 AM

Overall, there are common issues like:
* errors related to image, icon loading
* links working but not highlighted
* 'Title' from the markdown file contents aren't visible
* Videos aren't showing

### CODE REVIEW

File: app/cms/github.server.ts [hunk 1]
```diff
@@ -62,7 +63,14 @@ async function downloadMdxFileOrDirectory(
...
   fileOrDirPath: string
 ): Promise<{ entry: string; files: Array<GitHubFile> }> {
-  const mdxFileOrDirectory = `docs/${fileOrDirPath}`
+  /* If the repository is internal /flow content
+   * - Map Flow Content Paths to the right repository paths
+   * - Reassign repository to Flow
+   */
+  const mdxFileOrDirectory = flowContentNames.includes(repo)
+    ? `docs/content/${repo}/${fileOrDirPath}`
+    : `docs/${fileOrDirPath}`
```
[the remark this answers is not in the document]
Comment (10thfloor) [reply app/cms/github.server.ts:72]: Something like that...

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: app/cms/github.server.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (codingone21) [reply app/cms/github.server.ts:72]: This could work!

[the remark this answers is not in the document]
Comment (10thfloor) [reply app/cms/github.server.ts:72]: @codingone21 Any updates on this?

[the remark this answers is not in the document]
Comment (codingone21) [reply app/cms/github.server.ts]: Yep! Just refactored the code and pushed to this branch. The videos aren't rendering for me, but that might be related to the more recent changes on the main branch

File: app/routes/$repo.tsx [hunk 2]
```diff
@@ -15,25 +19,32 @@ type LoaderData = {
 export const loader: LoaderFunction = async ({
   params,
 }): Promise<LoaderData> => {
-  invariant(params.repo, `expected repo param`)
+  const content = params.repo
 
-  temporarilyRedirectToComingSoon()
```
Comment (codingone21) [inline app/routes/$repo.tsx:20]: should I keep this line `temporarilyRedirectToComingSoon()`  ? Not sure if we are going to push to prod as we continue developing

**Review by 10thfloor — APPROVED** [review]

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**